dns_server/src/main.rs

```diff
@@ -47,6 +47,9 @@ async fn run(config: Arc<DnsServerConfig>) -> Result<void::Void, error::DnsServe
         ping_timeout: Default::default(),
         node_type: NodeType::DnsServer.into(),
         allow_discover_private_ips: Default::default(),
+        header_count_limit: Default::default(),
```

Comment:
Sorry to comment on this again. But I suggest `msg_header_count_limit`

Reply:
Renaming is very easy thanks to the IDE support, so I don't mind renaming things any number of times if we can have better names as a result.